src/styles/styles.js

```diff
@@ -1092,20 +1092,20 @@ const styles = {
     },
 
     signInPageLeftContainer: {
-        paddingLeft: 40,
-        paddingRight: 40,
+        paddingLeft: 48,
+        paddingRight: 48,
     },
 
     signInPageLeftContainerWide: {
         maxWidth: variables.sideBarWidth,
     },
 
     signInPageWelcomeFormContainer: {
-        maxWidth: 300,
+        maxWidth: 279,
```

Comment:
Note for clean-up: I wonder if it might be good to make this a const, since it's tied to the hardcoded web button width